### ISSUE TITLE
Accessibility: bump indicator

### ISSUE THREAD

**Issue by iKarith** [mcgrew/dwrandomizer#126]
A simple enhancement to the game for deaf/HoH players: draw an on-screen indicator when the "bump" sound is played. Allows such players to more fairly bump their way through dark caves, if they can learn them. Also gives a quality of life benefit for tournaments and races since people who know their caves will be able to identify caves of the runner whose audio is not currently playing a bit more easily.

Suggested implementation: stone block map tile in an alternate color palette in the bottom right corner of the screen. This will tend not to interfere with displayed inputs or emulator functions and is out of the way when navigating. It could obscure the bottom corner 8x8 quarter-tile location during the instant of the *BUMP*, but say if that's the Rimuldar you're looking for or something, flashing from black and white to say green and back is likely to make it stand out more, not less.

**Comment by mcgrew** [comment]
This is a great idea but I'm not sure how hard this would be to implement. I'll look into it.

**Comment by knghtbrd** [comment]
I tried to suggest a method of drawing it that should be generally doable at any time: Draw a sprite you've got in a palette that makes it stand out like a sore thumb … the question is what's it take to hook in to the bUmP code, and how does one keep it on screen longer than a frame. My 6502 is a little weak, sadly. Deaf/HoH isn't usually the disability I'm looking to accommodate either, it just seemed like a useful idea in a race where audio wasn't available and a visual cue might help people follow dark cave dives.

Basically, if you stumble across a way to implement it, it'd be cool. 🙂

**Comment by odditude42** [comment]
can the routine used when enemies take damage to flash their sprites red be hooked into to flash the player sprite on a bonk?
